### ISSUE TITLE
Which version of Bootstrap do we want for D8?

### ISSUE THREAD

**Issue by soniktrooth** [drupalprojects/kalatheme#254]
Boostrap 4 is upcoming—do we want to try and get ahead of the curve here? 

http://blog.getbootstrap.com/2015/08/19/bootstrap-4-alpha/

**Comment by soniktrooth** [comment]
I feel like if we aren't expressly having to support Panopoly (something similar) we should probably aim high and run with v4. I can't see any downsides. 

I'd like to make a final decision on this before we get deep into template land so other's opinions/concerns here are welcome. 

Let's discuss @andrewmallis @labboy0276 @RobLoach

**Comment by soniktrooth** [comment]
also @derekderaps

**Comment by labboy0276** [comment]
It doesn't matter per say, we can build it for 3 trill out a version 4 and tweak the preprocesses and twigs and need be

That's what's super awesome about the new framework

**Comment by soniktrooth** [comment]
yeah, but why bother with 3 if nothing is stopping us going straight to 4? It's great that it will be able to handle both but It's not going to be an insignificant amount of work to do (and maintain) both 3 and 4. 

If 4 suits our needs going forward we might as well be on the latest version. If we need 3 for something specific we could implement it then, or possible someone else might want to contribute to a 3 version.

**Comment by labboy0276** [comment]
I will stare at the code for 4 

Right now I will do 3 since we have all the code for it already 

Path of least resistance right now

**Comment by pirog** [comment]
+1 for a robust framework loading framework (flf?) but lets not miss the forest through the trees; do the first implementation of kalatheme with the easiest and most immediately useful thing.

**Comment by RobLoach** [comment]
Bootstrap 4 has moved to SASS, so we will no longer need to use the unofficial/official source port for the custom builds that we do.

Understanding moving with the path of least resistance, it is futile. 4 has some syntactic differences that shouldn't block upgrading down the line.

**Comment by pirog** [comment]
BUT LESS IS THE FUTURE!!!!

**Comment by labboy0276** [comment]
We can have both in the future. Right now the code is there for BS 3.  I am just gonna truck ahead cause we have not tested any BS4 stuff and this is being used on a current site.

If it isnt that different syntactically, it will be easy to clone and assimilate.  That is the point of the new fresh and happier Kalatheme.

**Comment by hawkeyetwolf** [comment]
pfft, I’m going with 5.

**Comment by soniktrooth** [comment]
Understandable that @labboy0276 needs to run with a known entity for a real site so 3 it is. I'd really like to have 4 in beta so we can be cutting edge when other people start to look at it/use it.
